candies/cli/args/flag.py

```diff
@@ -14,9 +14,9 @@ class Flag:
     in the short-format.
 
     Attributes:
-        description (str): A description of an argument. Shown in help page.
-        short (str): A short form of an argument.
-            Must be non empty and consist of 1 letter.
+        description (str): A description of a flag. Shown in the help page.
+        short (str): A short form of a flag.
+            Must be non empty and consist of 1 alphabetic letter.
```

Comment:
Can letters be non-alphabetic?

Reply:
Not sure. Didn't see any opposite examples.

Right now the letter is checked using `isalpha()`, so I've just updated the corresponding comments.

Reply:
I mean, I would say that a letter is an alphabetic _symbol_. Thus, the alphabetic letter looks like a tautology to me.

Reply:
Oh, I see. Totally agree! I added `alphabetic`, because somehow though `1 letter` is not enough. Changed to `1 alphabet character` according to [this](https://www.geeksforgeeks.org/python-string-isalpha-application/) guide.